DEVELOPER_GUIDE.md

```diff
@@ -119,14 +128,14 @@ Let's suppose we have test-cases for put mapping and search api at first.
             └── put_mapping
                 ├── hooks.js
                 └── OpenSearchModel.json
-  ```
-
-We'd want to include the *Index-Aliases API* now.  
-  The project-tree structure will be as follows: 
-  ```
-  test
-  ├── scripts
-  └── model
+```
+
+We'd want to include the *Index-Aliases API* now.
+The project-tree structure will be as follows:
+```
+test
```

Comment:
[nit] the formatting of the directory tree seems to be a bit off, can you correct that?

Reply:
Corrected.